music21/style.py

```diff
@@ -105,7 +107,8 @@ def __init__(self) -> None:
     @property
     def enclosure(self) -> Enclosure|None:
         '''
-        Get or set the enclosure as a style.Enclosure enum or None.
+        Get or set the enclosure as a style.Enclosure enum (or
+        None, which means the enclosure is left unspecified).
 
         Valid names are:
```

Comment:
include literal None in the list below at the end.

At end of docs, mark as a "Changed in v9.7 -- ..."  as in other places in the docs.

Thanks!

Reply:
Done. (I also allowed the enclosure setter to take str, since the code actually handles that.)